### PR TITLE
feature/memoization_fibonacci

### PR DESCRIPTION
**Pull request by cdemiguel**

### CODE REVIEW

File: memoization/scripts/main_fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+console.log(fibonacci(10))
+
+///
+const fibonnacci = (number) => {
+    let cache = {};
+    let fiboArr = [];
+    let initialNumber0 = 0;
+
+    return (n) => {
+        if (n in cache) {
```
Comment (VGamezz19) [inline memoization/scripts/main_fibonacci.js]: Hey @cdemiguel All your features run right! but as I tried to tell you, if you want to see better the `memoization` you should try to do it with recursive function. But all okey, that an other way to do it!  good job 👍

**Comment by manuelbarzi** [comment]
NICE demi! please, move this challenge to your folder in staff/demi/fibonacci-memoization

**Comment by cdemiguel** [comment]
@manuelbarzi ,
Hiii!! manu i change the exercise on my folder ! i update the branch,
rewards!!

**Comment by manuelbarzi** [comment]
WELL done!

**Pull request merged**